doc/conf.py

```diff
@@ -126,14 +126,11 @@
     'travis_button': False,
     'codecov_button': False,
     'extra_nav_links': {
```

Comment:
Let's remove all these extra links in this PR. We'll put them all in the footer.